app/src/main/java/org/simple/clinic/onboarding/OnboardingScreen.kt

```diff
@@ -0,0 +1,64 @@
+package org.simple.clinic.onboarding
+
+import android.content.Context
+import android.text.style.TextAppearanceSpan
+import android.util.AttributeSet
+import androidx.constraintlayout.widget.ConstraintLayout
+import kotlinx.android.synthetic.main.screen_onboarding.view.*
+import org.simple.clinic.R
+import org.simple.clinic.util.Truss
+
+class OnboardingScreen(context: Context, attributeSet: AttributeSet) : ConstraintLayout(context, attributeSet) {
+
+  override fun onFinishInflate() {
+    super.onFinishInflate()
+    if (isInEditMode) {
+      return
+    }
+    setIntroTextViews()
+  }
+
+  private fun setIntroTextViews() {
+    val introOneFormattedString = Truss()
```

Comment:
Let's move setting individual intro text into its own methods since too much is happening in this method. WDYT?

Reply:
This doesn't seem to be addressed in this commit

Reply:
NVM, You have done it in a later commit.